google-cloud-bigquery-data_transfer/Gemfile

```diff
@@ -8,7 +8,3 @@ gem "google-cloud-errors", path: "../google-cloud-errors"
 gem "google-cloud-bigquery-data_transfer-v1", path: "../google-cloud-bigquery-data_transfer-v1"
 
 gem "rake"
```

Comment:
Actually, can you remove this line, and add `"rake", ">= 12.0"` as a development dependency in the gemspec? This will match what the generator will be generating after https://github.com/googleapis/gapic-generator-ruby/pull/432

Reply:
So that when the generator reruns, we won't get another trivial change.

Reply:
Thanks, done!